xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -141,6 +141,7 @@ bool CRenderManager::Configure(const VideoPicture& picture, float fps, unsigned
     m_dvdClock.SetVsyncAdjust(0);
     m_pConfigPicture.reset(new VideoPicture());
     m_pConfigPicture->CopyRef(picture);
+    UpdateLatencyTweak();
```

Comment:
wrong method. I meant the configure without arguments (that's why I explicitly included the braces in my comment)
this one here is called form player thread, the right one from render thread.

but it is not needed anyway:
https://github.com/xbmc/xbmc/blob/cf024792cd3bf801df4f83e37af308cc4cb1961c/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp#L219

Configure() already sets m_bTriggerUpdateResolution, which results in a call to UpdateLatencyTweak()

p.s.
Your first attempt was ok but the comment resulted in this extra round :)

Reply:
Thx for explaining the reasons why the attempt was wrong @FernetMenta , PR updated.